src/main/resources/openapi/schemas/reportUseOverTimeJournal.json

```diff
@@ -0,0 +1,45 @@
+{
+  "description": "use over time report for a journal",
+  "type": "object",
+  "properties": {
+    "kbTitleId": {
+      "type": "string",
+      "description": "kb title identifier"
+    },
+    "kbTitleName": {
```

Comment:
Rename to "title", rephrase description to "Title of the item"

Reply:
Changed.